lightningd/channel_control.c

```diff
@@ -93,6 +93,11 @@ static void try_update_feerates(struct lightningd *ld, struct channel *channel)
 	if (!channel->owner)
 		return;
 
+	/* The feerate message is only understood by `channeld` so
+	 * don't attempt to send it to other subds*/
+	if (!streq(channel->owner->name, "channeld"))
+		return;
+
 	channel_update_feerates(ld, channel);
```

Comment:
looks like you are building on an already in master commit?

Reply:
Sorry, fixed.